tests/wpt/metadata/dom/nodes/Document-contentType/contentType/contenttype_javascripturi.html.ini

```diff
@@ -1,5 +1,5 @@
 [contenttype_javascripturi.html]
   type: testharness
   [Javascript URI document.contentType === 'text/html']
-    expected: FAIL
+    expected: TIMEOUT
```

Comment:
Those tests timed out without our changes. We ran our tests on a unmodified servo code.